README.md

```diff
@@ -57,6 +57,57 @@ all the dependencies automatically.
 *   [docker >= 19.03.5](https://www.docker.com/)
 *   [graphviz](https://graphviz.gitlab.io/download/)
 
+## Setting up local machine
+1. Consent Manager needs a bunch of backends like RabbitMQ, Keycloak, Postgres.  These backends also need some initialization, 
+e.g. 2 Realms (consent-manager & central-registry) & a user *consent-service-admin-user* need to be created for user authentication in Keycloak.  Also, relevant databases & schemas need to be created in postgresql.
+Run the following docker-compose command for initializing the backend
+
+```alpha
+docker-compose -f docker-compose-backend.yml up
+```  
+Leave the terminal window for these backends to be running.
+
+Start and stop the backend using the following commands
+```alpha
+#Stop backends
+docker-compose -f docker-compose-backend.yml stop
+#Start backends
+docker-compose -f docker-compose-backend.yml start
+```  
+
+
+If you want to remove all the backends for good
+
+```alpha
+docker-compose -f docker-compose-backend.yml down
+docker volume rm consent-manager_postgres_data
+```
+
+2. With the above setup, you'll have all the required database created.  Now run the database migrations to create the required schema for consent-manager app.
+
+
+```alpha
+git clone https://github.com/ProjectEKA/cm-db-initializer
+mvn clean install
+java -Djdbc.url=jdbc:postgresql://localhost:5432/consent_manager -Djdbc.username=postgres -Djdbc.password=password -jar target/cmdb-initializer-1.0-SNAPSHOT.jar
+docker volume rm consent-manager_postgres_data
+```
+  
+3. After you Keycloak setup, you'll need to keep track of the Secret associated to consent-manager 
+* Navigate to ConsentManager Realm
+* Go to "Clients" > "Credentials"
+* Get the Secret field
+* Update the secret in [docker-compose.yml](docker-compose.yml) under variables CONSENTMANAGER_CLIENTREGISTRY_XAUTHTOKEN & CONSENTMANAGER_KEYCLOAK_CLIENTSECRET 
```

Comment:
* CONSENTMANAGER_CLIENTREGISTRY_XAUTHTOKEN => this one needs to be copied from central-registry realm
* CONSENTMANAGER_KEYCLOAK_CLIENTSECRET => this one needs to be copied from ConsentManager realm

Reply:
Got it.. Will fix the readme

Reply:
Done